src/app/auth/profiles/profiles.component.ts

```diff
@@ -147,24 +113,6 @@ export class ProfilesComponent implements OnInit {
     this.errorDialog.open(JsonParseErrorDialogComponent);
   }
 
-  /**
-   * Verifies that every profile is correctly defined in the array of profiles.
-   * @param profiles - Array of profiles sourced from profiles.json
-   */
-  assertProfilesValidity(profiles: Profile[]): void {
-    if (profiles.filter(profile => (profile.profileName === undefined || profile.encodedText === undefined)).length !== 0) {
-      throw new Error();
-    }
-  }
-
-  /**
-   * Returns true if the file indicated by the filePath exists.
-   * @param filePath - Path of file to check.
-   */
-  userProfileFileExists(filePath: string): boolean {
-    return this.electronService.fileExists(filePath);
```

Comment:
Shall we also remove the methods in `ElectronService` which were used for automatic profile detection?
(`fileExists`, `readFile`, `getCurrentDirectory`)

We could also remove the main Electron process code associated with `getCurrentDirectory`

Reply:
Yes sure 👍